packages/beacon-node/src/chain/clock/LocalClock.ts

```diff
@@ -14,7 +14,7 @@ import {IBeaconClock} from "./interface.js";
 export class LocalClock implements IBeaconClock {
   private readonly config: IChainForkConfig;
   private readonly genesisTime: number;
-  private timeoutId: NodeJS.Timeout;
+  private timeoutId: number | NodeJS.Timeout;
```

Comment:
Allows to compile with any recent @types/node version. Has caused issues when installing individual local packages like c-kzg